packages/block-editor/src/components/inserter/menu.js

```diff
@@ -6,18 +6,12 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import {
-	forwardRef,
-	useState,
-	useCallback,
-	useMemo,
```

Comment:
I wonder if its ok to remove all the useMemos

Reply:
I'm not sure -- let's performance test it and see if it's the same or better without it, and then we can add it back in if we need to. Before it was especially necessary because everything was running everything on mount. Now it doesn't build all at once but each tab content gets rendered when it's clicked.

Reply:
I tried a memoized and unmemoized version of the blocks tab, and memoizing was faster by our specs:

memo:
│ inserterOpen   │ '15.22 ms' │
│ inserterSearch │ '4.79 ms'  │
│ inserterHover  │ '1.83 ms'  │


no memo
│ inserterOpen   │ '15.76 ms' │
│ inserterSearch │ '5.56 ms'  │
│ inserterHover  │ '1.89 ms'  │